src/backend/distributed/commands/create_citus_local_table.c

```diff
@@ -111,6 +112,17 @@ create_citus_local_table(PG_FUNCTION_ARGS)
 }
 
 
+/*
+ * create_citus_local_table is a wrapper function for old name of
+ * of citus_add_local_table_to_metadata.
+ */
+Datum
+create_citus_local_table(PG_FUNCTION_ARGS)
```

Comment:
We have never exposed `create_citus_local_table ` to the users. So, instead, can we simply give a ERROR saying that use  `citus_add_local_table_to_metadata `?

As you noted we had `create_citus_local_table` on the change log of 9.5, still we never showed in docs/blogs etc. And, with Citus 10, it should be OK to break some users.

Maybe add one line of extra `DESCRIPTION: Drops support for  create_citus_local_table  in favor of citus_add_local_table_to_metadata ` or such

Reply:
This function only exists to support testing 9.5->10.0 downgrade with a 10.0 binary. It does not matter much what we do here, since it should never be called in practice.

Reply:
True, but we have a test in those downgrade steps that first creates a citus local table in citus 9.5 and we error out if we try downgrade back to 9.4, so I think we still need that udf in binary.
Or, I can remove that test too but not sure if this is a good option.

Reply:
I'd at least add a notice that `create_citus_local_table ` is deprecated and use  `citus_add_local_table_to_metadata ` instead